packages/legend-graph/src/graph/metamodel/pure/dataProduct/DataProduct.ts

```diff
@@ -59,6 +60,10 @@ export class LakehouseAccessPoint extends AccessPoint {
     super(id);
     this.targetEnvironment = targetEnv;
     this.func = func;
+
+    makeObservable(this, {
+      description: observable,
```

Comment:
this is not the correct place for `observable`. we have graph observer methods where this goes.